README.md

```diff
@@ -25,7 +22,7 @@ Notes:
 
 ## Installation
 
-This package is available on [npm][npm-url] as: `extend`
+This package can be installed using a package manager like npm, yarn, or bun:
```

Comment:
this isn’t valuable; and the original sentence is correct, since all these clis install from npm

Reply:
Sounds good, restored the original and committed the other requested change